### PR TITLE
[FEATURE] Add newer compiler to travis CI.

### PR DESCRIPTION
**Pull request by rrahn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,22 +38,22 @@ matrix:
...
-#    # clang-3.9 currently not white listed.
-#    - os: linux
-#      compiler: clang-3.9
-#      addons:
-#        apt:
-#          sources: ['ubuntu-toolchain-r-test', 'george-edison55-precise-backports' , 'llvm-toolchain-precise-3.9']
-#          packages: ['clang-3.9', 'g++-5', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip'] # g++ required for newer libstdc++
-#      install: export CXX="clang++-3.9"
+          sources: ['ubuntu-toolchain-r-test', 'george-edison55-precise-backports' , 'llvm-toolchain-precise-3.9']
+          packages: ['clang-3.9', 'g++-5', 'cmake', 'cmake-data', 'zlib1g-dev', 'libbz2-dev', 'libboost-dev', 'python', 'python-nose', 'python-jinja2', 'python-pip'] # g++ required for newer libstdc++
```
Comment (marehr) [inline .travis.yml]: Why don't you use a newer g++ version?

**Comment by rrahn** [comment]
!rebuild

**Comment by rrahn** [comment]
I check if only allowing one executor per machine is fixing the connection interrupt issue with the windows nodes.

**Comment by marehr** [comment]
Apparently not? Lets merge it.

**Comment by h-2** [comment]
merge?

**Comment by marehr** [comment]
@h-2 👍

**Pull request merged**